backend/src/v5/services/eventsListener/components/modelEvents.js

```diff
@@ -164,8 +168,17 @@ const revisionUpdated = async ({ teamspace, project, model, data, sender, modelT
 		[modelTypes.DRAWING]: chatEvents.DRAWING_REVISION_UPDATE,
 	};
 
-	await createModelMessage(modelEvents[modelType], { ...data, _id: UUIDToString(data._id) },
-		teamspace, project, model, sender);
+	let calibration;
+	if (modelType === modelTypes.DRAWING) {
```

Comment:
I dont' think we need clibration here. the only situation this is trigged right now is if something is set to void i think?

calibration wouldn't have changed and shouldn't be in the dataset.